tests/behat/features/capabilities/language/change-language.feature

```diff
@@ -49,7 +49,8 @@ Feature: Multilingual Open Social
     # Switch to Dutch.
     When I select "Dutch" from "Interface language"
     And I press "Save"
-    Then I should see the text "Lokalisatie-instellingen"
+#    Then I should see the text "Lokalisatie-instellingen"
```

Comment:
@OH17L I guess these commented test steps can be deleted right?

Reply:
@bramtenhove, no, it is not. I did it because current Drupal core does not have a translation for this phrase. When a translation of this phrase will be restored then this line is uncommented and next line is deleted.